modules/binary-authorization/variables.tf

```diff
@@ -34,3 +34,15 @@ variable crypto-algorithm {
   default     = "RSA_SIGN_PKCS1_4096_SHA512"
   description = "Algorithm used for the async signing keys"
 }
+
+variable "disable_services_on_destroy" {
+  description = "Whether project services will be disabled when the resources are destroyed. https://www.terraform.io/docs/providers/google/r/google_project_service.html#disable_on_destroy"
+  default     = "false"
```

Comment:
These should be booleans.

Reply:
Done. I was thinking about making them bool but kept it string as these variables are defined as string in sub-module project_services

https://github.com/terraform-google-modules/terraform-google-project-factory/tree/master/modules/project_services